pkgs/development/python-modules/black/default.nix

```
@@ -71,7 +74,7 @@ buildPythonPackage rec {
     typed-ast # required for tests and python2 extra
     uvloop
   ] ++ lib.optional (pythonOlder "3.7") dataclasses
```

```suggestion
  ]
```
We don't have Python 3.6 anymore.